bzt/modules/console.py

```diff
@@ -114,8 +114,9 @@ def prepare(self):
         if isinstance(self.engine.aggregator, ResultsProvider):
             self.engine.aggregator.add_listener(self)
 
-        self.disabled = self.settings.get("disable", False)
-        if self.disabled:
+        disable = str(self.settings.get('disable', 'auto')).lower()
+        if (disable == 'true') or ((disable == 'auto') and (not is_tty())):
```

Comment:
I can't agree with this. So console will be disabled if disable='auto' and stdout is tty. Yes, we can add '==True' at the end of this line... but I don't like it too much. At my opinion saving different types in one variable isn't good practice.